docs/tool/js/views/map-view.js

```diff
@@ -396,37 +425,43 @@ var mapView = {
              }
         });
         mapView.map.on('click', function (e) {
+            console.log(e);
             var building = (mapView.map.queryRenderedFeatures(e.point, { layers: ['project'] }))[0];
+            console.log(building);
             if ( building === undefined ) return;
-            setState('previewBuilding',{
-                proj_addre: building.properties.proj_addre,
-                proj_name: building.properties.proj_name,
-                hud_own_name: building.properties.hud_own_name
-            });
+            setState('previewBuilding', building);
+
+                /*{ 
```

Comment:
Thanks! Oh yeah -- the commented out code can be removed. I needed it for references, but no longer.